components/atom/tooltip/src/index.scss

```diff
@@ -221,6 +200,20 @@ $c-atom-tooltip-colors: (
     }
   }
 
+  &[data-placement='left-start'],
+  &[data-placement='right-start'] {
+    #{$class-inner} {
+      transform: translateY(-8px);
```

Comment:
It is better to use variables for this `translateY` value.

Reply:
removed